packages/braintree-integration/src/get-valid-button-style.ts

```diff
@@ -18,17 +18,24 @@ export default function getValidButtonStyle(style: PaypalStyleOptions): PaypalSt
     return omitBy(validStyles, isNil);
 }
 
-function getValidHeight(height?: number): number {
+function getValidHeight(height: number | string): number {
     const minHeight = 25;
+    const defaultHeight = 40;
     const maxHeight = 55;
 
-    if (typeof height !== 'number' || height > maxHeight) {
+    const currentHeight = typeof height === 'string' ? +height : height;
```

Comment:
can be simplified

`const currentHeight = +height;` or `const currentHeight = Number(height);`